spec/requests/users_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User do
```

Comment:
- [ ] As explained in my previous comment. Please add the type request to this line.